.github/workflows/check.yml

```diff
@@ -0,0 +1,106 @@
+name: "Check Workflow"
+
+on: [push, pull_request]
+
+jobs:
+
+  perl-versions:
+    runs-on: ubuntu-latest
+    name: 'perl-versions default'
+    outputs:
+      perl-versions: ${{ steps.action.outputs.perl-versions }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: "uses perl-versions"
+        id: action
+        uses: ./
+
+  since-v520:
+    runs-on: ubuntu-latest
+    name: 'since v5.20'
+    outputs:
+      perl-versions: ${{ steps.action.outputs.perl-versions }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: "uses perl-versions"
+        id: action
+        uses: ./
+        with:
+          since-perl: v5.20
+
+  since-520:
```

Comment:
is this duplicated entry intended or mistake?

Reply:
this intended to ensure we support both flavor `v5.20` and `5.20` might be useless

Reply:
oh, right, `v`, missed that